example/python/irohalib.py

```diff
@@ -211,6 +217,28 @@ def query(self, name, counter=1, creator_account=None, created_time=None, **kwar
             internal_query.CopyFrom(message)
         return query_wrapper
 
+    def blocks_query(self, counter=1, creator_account=None, created_time=None):
```

Comment:
May be just ```creator_account=self.creator_account, created_time=self.now()```? Lines 228-231 could be removed after that.

Reply:
Unfortunately, that won't work, since self is not defined at the moment of evaluation.

```
>>> class A(object):
...     def __init__(self):
...             self.a = 'abc'
...     def x(self, param=self.a):
...             print(param)
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 4, in A
NameError: name 'self' is not defined
```

Reply:
Python was definitely overrated by me.